test/test-examples-server-custom-bindings.js

```diff
@@ -0,0 +1,46 @@
+/*jshint expr: true*/
+"use strict";
+var chai = require("chai");
+var expect = chai.expect;
+chai.config.includeStack = true;
+var request = require("supertest-as-promised");
+var alexaAppServer = require("../index");
+
+// used so that testing https requests will work properly
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
I'll merge, however it feels like this should be set per-test, or in a `beforeEach` and then reset back in an `afterEach`.